test.sh

```diff
@@ -1,7 +1,7 @@
 #!/bin/bash
 
-local file="$Ash__ACTIVE_MODULE_DIRECTORY/extra/example.yaml"
-local prefix="YamlParse_output_"
+declare file="$Ash__ACTIVE_MODULE_DIRECTORY/extra/example.yaml"
+declare prefix="YamlParse_output_"
```

Comment:
@mcstafford-git - I've only ever used declare with flags, and to my understanding is redundant without any.

But yes, these definitely should not have been local!